### PR TITLE
minor: improve output of inspections in CI

### PR DESCRIPTION
**Pull request by nrmancuso**
Improves output of IDEA inspections in CI

Link to inspections workflow: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/17868/workflows/bad7f959-2fad-4a94-a632-6ec27c2c1d61/jobs/265571

Output:
```
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  19.023 s
[INFO] Finished at: 2023-03-23T04:36:29Z
[INFO] ------------------------------------------------------------------------

####################################################################################################
Intellij Idea validation is about to start
Progress output will be flushed at end. Validation is in progress ...
Checking results ...

ATTENTION GITHUB CONTRIBUTOR:
There are inspection problems.
Review results in 'ARTIFACTS' tab in the CircleCI UI above, or follow the link below:
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/17861/workflows/bad7f959-2fad-4a94-a632-6ec27c2c1d61/jobs/265571/artifacts

Exited with code exit status 1
CircleCI received exit code 1

```

### CODE REVIEW

**Comment by nrmancuso** [comment]
@shubh220922 please take a look, let me know if this is change is helpful

**Comment by shubh220922** [comment]
@nrmancuso 

Awesome. Error was easily traceable.
Error message could easily be found. 
image

File: .ci/idea-inspection.sh [hunk 1]
```diff
@@ -49,10 +55,12 @@ echo "Checking results ..."
...
 
 if [[ $PROBLEM_COUNT -gt 0 ]] && [[ "$CIRCLECI" == "true" ]]; then
-    echo -e "\n\n\n\n"
+    echo ""
     echo "ATTENTION GITHUB CONTRIBUTOR:"
     echo "There are inspection problems."
-    echo "Review results in 'ARTIFACTS' tab in the CircleCI UI above."
+    echo "Review results in 'ARTIFACTS' tab in the CircleCI UI above, or follow the link below:"
+    CIRCLECI_APP_URL="https://app.circleci.com/pipelines/github/checkstyle/checkstyle"
+    echo "$CIRCLECI_APP_URL/17861/workflows/$CIRCLE_WORKFLOW_ID/jobs/$CIRCLE_BUILD_NUM/artifacts"
```
Comment (romani) [inline .ci/idea-inspection.sh:63]: >17861

What is this? Can it be different?

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/idea-inspection.sh (hunk 1 above)
Comment (nrmancuso) [reply .ci/idea-inspection.sh:63]: I tried different numbers in a few different PRs (my own, and others) and it still worked. It seems like this number doesn’t actually matter. Example: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/1/workflows/e75ebb44-1f20-4e32-84a9-3fa3215872a9/jobs/265866/artifacts

https://app.circleci.com/pipelines/github/checkstyle/checkstyle/17/workflows/e75ebb44-1f20-4e32-84a9-3fa3215872a9/jobs/265866/artifacts

Comment (romani) [reply .ci/idea-inspection.sh:63]: if does not matter, let make it `1` as clear sign that for now CirlceCi ignore this

Comment (nrmancuso) [reply .ci/idea-inspection.sh:63]: Done, left a comment as well.

**Pull request merged**